go/host/enclave/guardian.go

```diff
@@ -645,21 +649,25 @@ func (g *Guardian) periodicBundleSubmission() {
 				continue
 			}
 
-			if from.Uint64() > fromSequenceNumber {
-				fromSequenceNumber = from.Uint64()
+			if from.Uint64() > fromSequenceNumber.Uint64() {
+				fromSequenceNumber.Set(from)
 			}
 
-			bundle, err := g.enclaveClient.ExportCrossChainData(context.Background(), fromSequenceNumber, to.Uint64())
+			bundle, err := g.enclaveClient.ExportCrossChainData(context.Background(), fromSequenceNumber.Uint64(), to.Uint64())
 			if err != nil {
 				if !errors.Is(err, errutil.ErrCrossChainBundleNoBatches) {
 					g.logger.Error("Unable to export cross chain bundle from enclave", log.ErrKey, err)
 				}
+				if errors.Is(err, context.DeadlineExceeded) {
```

Comment:
so this is the fix for the load issue?

Reply:
No, this is just a patch to shutdown on UAT and Dev testnet if we ever hit it